war/pom.xml

```diff
@@ -733,7 +733,7 @@ THE SOFTWARE.
                                 <goal>gulp</goal>
                             </goals>
                             <configuration>
-                                <arguments>bundle</arguments>
+                                <arguments>bundle lint</arguments>
```

Comment:
@kzantow Sure! Then we have to do the same here https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L1102